### PR TITLE
API call to obtain locales list

### PR DESCRIPTION
**Pull request by erickskrauch**

### CODE REVIEW

**Comment by kdzwinel** [comment]
Thank you for contribution! Code looks 👍 Before we merge though, could you please add two things:

- tests - https://github.com/brainly/nodejs-onesky-utils/tree/master/test
- documentation - https://github.com/brainly/nodejs-onesky-utils#example , https://github.com/brainly/nodejs-onesky-utils#api

**Comment by erickskrauch** [comment]
NP, will soon be.

**Comment by kdzwinel** [comment]
@erickskrauch hey! Do you still want to finish this?

**Comment by erickskrauch** [comment]
@kdzwinel, yes of course. Sorry for the delay.

**Comment by erickskrauch** [comment]
Well, I almost did not forget :)

File: README.md [hunk 1]
```diff
@@ -67,6 +67,24 @@ onesky.postFile(options).then(function(content) {
 });
 ```
 
+### postFile
```
Comment (kdzwinel) [inline README.md]: that should probably say "getLanguages" ;)

Comment (erickskrauch) [inline README.md]: Fixed :)

**Comment by erickskrauch** [comment]
@kdzwinel can it be merged?

**Comment by kdzwinel** [comment]
@SebastianCzoch can you please merge it and deploy it?

**Comment by SebastianCzoch** [comment]
Hey @erickskrauch, I've released a new version with your changes. https://www.npmjs.com/package/onesky-utils

Thanks for your contribution!

**Pull request merged**